src/Trains.NET.Engine/Utilities/Random/BasicPRNG.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Trains.NET.Engine;
+
+public struct BasicPRNG
+{
+    public int Seed;
+    private const int Multiplier = 16807; // Random-ish
+    private const int Max = 2147483647; // Tested-ish
+    private const float MaxFloat = Max;
+
+    public BasicPRNG(int seed)
+    {
+        Seed = seed;
+    }
+
+    public int Next() => Seed = Math.Abs(Seed * Multiplier % Max);
```

Comment:
lol, classic scott. Is this actually better than System.Random? 😛

Oh actually.... they changed Random in .NET 6.. maybe they broke its pseudo-ness

Reply:
Yep, classic Scott, wanted to see if this would work, and it turns out it works really well!

Reply:
It's fast though 😄 
![image](https://user-images.githubusercontent.com/20675893/140508868-cfa45c3d-90a5-41d7-bf41-0be407e64a0b.png)
(especially as we are using it less than 100 times 😉)